pom.xml

```diff
@@ -123,10 +123,10 @@
         <jknack.handlebars.version>2.0.0</jknack.handlebars.version>
         <marionette.version>2.4.5</marionette.version>
         <moment.version>2.20.1</moment.version>
-        <node.version>v8.9.4</node.version>
+        <node.version>v10.16.0</node.version>
```

Comment:
Does the `path.sh` script need to be updated?

Reply:
It should pick up these version automatically :+1: